.buildkite/build_vaxine.yml

```diff
@@ -9,4 +10,11 @@ agent:
 steps:
   - label: ":whale: Build & push the vaxine container"
     command: make buildkite-build-and-push
-
+  - wait
+  - label: ":rocket: Publish the image to DockerHub"
+    if: build.tag =~ /^[0-9]+\.[0-9]+\.[0-9]+/
```

Comment:
We already take IMAGE_TAG in makefile. Why do we want to be extra protective here?

Reply:
This second job is run with a different target repository. We don't want to push to DockerHub unless it was from a new Git tag. This also answers why env variables live here - I don't want to mix deployment specifics (i.e. target repos) into Makefile when it's orchestrated by the CI